src/components/accountForm/accountForm.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CustomModal } from '../shared/Modal/Modal';
+import SignIn from './SignIn/SignIn';
+import SignUp from './SignUp/SignUp';
+import Button from 'react-bootstrap/Button';
+import './accountForm.css';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLongArrowAltLeft } from '@fortawesome/free-solid-svg-icons';
+
+const AccountForm = (props) => {
+    return (
+        <CustomModal {...props} size='lg' aria-labelledby='contained-modal-title-vcenter' centered>
```

Comment:
add state that controls CustomModal visibility. this state can be passed from props